cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/cardverifyui/CardVerifyActivity.kt

```diff
@@ -247,24 +248,21 @@ open class CardVerifyActivity : SimpleScanActivity<RequiredCardDetails>() {
             }
             null
         }
-    } ?: RequiredCardDetails(null, "1234") // TODO: this is a hack
+    }
 
     private fun onScanDetailsAvailable(
         requiredCardDetails: RequiredCardDetails?,
     ) {
-        if (requiredCardDetails == null) {
-            scanFailure(InvalidRequiredCardException("Missing last four"))
-            return
+        if (requiredCardDetails != null) {
```

Comment:
The intention of this PR is to make these details optional in the case that a user is scanning a card where no details have been provided prior to the scan (e.g. the user is adding a card to their account). Because this is now a supported use case, we do not want to fail.